### PR TITLE
Automatically check additional views for memory leaks on navigation

### LINKED ISSUE #6734: Automatically Check Additional Views for Memory Leaks on navigation
Follow on from https://github.com/nasa/openmct/issues/4406

https://github.com/nasa/openmct/pull/6718 introduces the ability to test views for memory leaks on navigation, as well as memory leak tests for a number of screens. To limit the scope of that PR we have split the screens between two issues. This issue contains the second and final batch of screens that need memory leak tests defined in the open repo.

Note that tests are defined for _some_ but not all of these, but are disabled due to finding memory leaks. The memory leaks will need to be fixed before merging this PR.

* [x] Gauges
* [x] Plan View
* [x] Time List View
* [x] Scatter Plot
* [x] Graph
* [x] Gantt Chart
* [x] Clocks
* [x] Timers
* [x] Web Page
* [x] Complex Display Layouts

### PR DESCRIPTION
**Pull request by scottbell**
Closes #6734 

### Describe your changes:

- Add navigation memory leak checks for more components.
- Cleanup plotly when Vue component is dismounted.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [x] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Review by ozyx — APPROVED** [review]
LGTM! Refactored the suite with some better locators and got rid of some redundant e2e code

**Comment by unlikelyzero** [comment]
@scottbell you have a failure on restricted notebook

**Comment by ozyx** [comment]
> @scottbell you have a failure on restricted notebook

Lemme make sure it's not due to my changes.

**Comment by ozyx** [comment]
> > @scottbell you have a failure on restricted notebook
> 
> Lemme make sure it's not due to my changes.

It's a known flake. on the perf test, I just re-ran the plot view test 5x and no flakes on my end (in ci env), so it might be a very rare flake?

File: src/plugins/charts/bar/inspector/BarGraphOptions.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@
       <li>
         <series-options
           v-for="series in plotSeries"
-          :key="series.key"
+          :key="series.keyString"
```
[the remark this answers is not in the document]
Comment (scottbell) [reply src/plugins/charts/bar/inspector/BarGraphOptions.vue:29]: this was your catch!

**Pull request merged**